src/main/java/adris/altoclef/tasks/ResourceTask.java

```diff
@@ -176,6 +176,17 @@ protected Task onTick(AltoClef mod) {
             }
         }
 
+        return onResourceTick_clearingPhase(mod);
+    }
+
+    private Task onResourceTick_clearingPhase(AltoClef mod) {
```

Comment:
At this rate we should probably just move this logic up to `onTick`

Reply:
done